pkg/kubelet/kubelet_network.go

```diff
@@ -67,5 +67,5 @@ func (kl *Kubelet) updatePodCIDR(ctx context.Context, cidr string) (bool, error)
 // This function is defined in kubecontainer.RuntimeHelper interface so we
 // have to implement it.
 func (kl *Kubelet) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error) {
-	return kl.dnsConfigurer.GetPodDNS(pod)
+	return kl.dnsConfigurer.GetPodDNS(context.TODO(), pod)
```

Comment:
Please, add a comment explaining why this is used and when it must be replaced.

Reply:
@zhifei92 ping

Reply:
Done, sorry for the delayed response.